bingo.js

```diff
@@ -240,38 +255,50 @@ function loadSettings()
 {
 	getSettingsFromURL();
 	getSettingsFromLocalStorage();
+	checkRecentCurrentSheet();
+	loadProgressFromURL();
```

Comment:
Both functions `checkRecentCurrentSheet` and `loadProgressFromURL` check the value `gup(URL_PARAM_PROGRESS)`. Code would be more readable if it was checked here to only call one of the functions.